pages/_error.tsx

```diff
@@ -0,0 +1,52 @@
+// CSS
+import styles from '../styles/Error.module.scss';
+// Components
+import NavBar from '../components/NavBar';
+import Button, { ButtonType } from '../components/Button';
+
+export default function ErrorPage({ 
+  statusCode 
+}:{
+  statusCode: any
+}) {
+  const returnTo = '/';
+
+  return (
+    <div className={styles.canvas}>
```

Comment:
small nit can we change the name `canvas` it can be a bit confusing given there is an html canvas element. maybe something make this `container` and change the container class to be `content` or `pageContent`.